packages/ui/src/lib/layouts/Page.svelte

```diff
@@ -1,23 +1,54 @@
 <script lang="ts">
-import { createEventDispatcher, onDestroy, onMount } from 'svelte';
+import { onDestroy, onMount, type Snippet } from 'svelte';
 
 import { LinearProgress } from '..';
 import CloseButton from '../button/CloseButton.svelte';
 import Link from '../link/Link.svelte';
 
-export let title: string;
-export let titleDetail: string | undefined = undefined;
-export let subtitle: string | undefined = undefined;
-export let breadcrumbLeftPart: string | undefined = undefined;
-export let breadcrumbRightPart: string | undefined = undefined;
-export let breadcrumbTitle: string | undefined = '';
-export let hasClose: boolean = true;
-export let inProgress: boolean = false;
+interface Props {
+  title: string;
+  titleDetail?: string;
+  subtitle?: string;
+  breadcrumbLeftPart?: string;
+  breadcrumbRightPart?: string;
+  breadcrumbTitle?: string;
+  hasClose?: boolean;
+  inProgress?: boolean;
+  onclose?: () => void;
+  onbreadcrumbClick?: () => void;
+  icon?: Snippet;
+  subtitleSnippet?: Snippet;
+  actions?: Snippet;
+  detail?: Snippet;
+  tabs?: Snippet;
+  content?: Snippet;
+}
+
+const {
+  title,
+  titleDetail = undefined,
+  subtitle = undefined,
+  breadcrumbLeftPart = undefined,
+  breadcrumbRightPart = undefined,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
can be removed if https://github.com/podman-desktop/podman-desktop/pull/12559 is merged